docker/monitoring/prometheus/prometheus.yml

```diff
@@ -0,0 +1,5 @@
+scrape_configs:
+  - job_name: 'raiko'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['host.docker.internal:8080']
```

Comment:
Is this the correct target even when not running raiko inside the docker? Just checking because I can connect to `http://localhost:8080/metrics` but not this one, but I don't know how this docker/not in docker thing works.

Reply:
For me, it wasn't connecting via `localhost:8080` because it was calling the inner prometheus host's localhost, this was querying the local machine itself, but I'll check if this is still correct.